tests/test_card.py

```diff
@@ -48,6 +48,18 @@ def test_all_with_params_return_cards(self):
                         .all()
             
             self.assertTrue(len(cards) >= 13)
+
+    def test_same_request_twice(self):
+        # this is used to test cache
+        with vcr.use_cassette('fixtures/legendary_elf_warriors.yaml'):
+            cards = Card.where(supertypes='legendary') \
+                .where(subtypes='elf,warrior') \
+                .all()
+            self.assertTrue(len(cards) >= 13)
+            cards = Card.where(supertypes='legendary') \
+                .where(subtypes='elf,warrior') \
+                .all()
+            self.assertTrue(len(cards) >= 13)
```

Comment:
I know, this is just a basic sanity check to make sure that the cache doesn't break anything.
Multicache module itself has its own tests